lib/ts/recipe/emailverification/index.ts

```diff
@@ -159,20 +165,23 @@ export default class Wrapper {
         tenantId: string,
         token: string,
         attemptAccountLinking: boolean = true,
-        userContext: any = {}
+        session?: SessionContainerInterface,
```

Comment:
If the session user is not verified but linking requires verification, in most cases we'd redirect the user to the EV screen when they try to add a pw because we can't make the session user primary. 
They can then come back and enter the password.

If they were already primary however we return an error code to the user and not link. We do not expect this flow to go through EV because we want the user to only be able to trigger the email verification flow for the current user.

This is not unsolvable, we could do it in the future through the EV flow, but the recommendation would be to do it through OTP.

Reply:
All in all, I'll remove the session input from this function, however that's going to be a tiny bit strange since that means that the account linking functions will be called without a session even though it's there.

Reply:
Thats ok. Cause we don't want to do MFA in that context anyway.